source/_includes/custom/features.html

```diff
@@ -95,4 +95,24 @@
       </a>
     </div>
   </div>
+    <div class="card">
+    <div class="card-header">
+      <i class="icon-sun"></i>
+      Home Energy Management
+    </div>
+    <div class="card-content">
+      <p>
+        Home Assistant allows you to get on top of your energy use with its 
+        home energy management feature. Gain new insights, 
+        optimize your solar panel production, plan energy usage and save money.
+      </p>
+
+      <a
+        class="button"
+        href="/docs/energy"
```

Comment:
Add trailing slash to avoid redirect?

Reply:
```suggestion
        href="/docs/energy/"
```
Right, forgot we have that pagerule